app/services/widgets/widgets-data.ts

```diff
@@ -682,4 +668,13 @@ export const WidgetDisplayData = (platform?: string): { [x: number]: IWidgetDisp
     platforms: new Set(['twitch']),
     icon: 'icon-community',
   },
+  [WidgetType.GameWidget]: {
+    name: $t('Game Widget'),
+    description: $t('Let your viewers play a game in chat'),
+    demoVideo: false,
+    demoFilename: 'game-widget.png',
```

Comment:
Are these hooked up to the CDN fallback / are these up on the CDN yet?

Reply:
yep!